src/common/ZSwitch.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Switch } from 'react-native';
+import { BRAND_COLOR } from './styles';
+
+export default class ZSwitch extends React.PureComponent {
```

Comment:
I don't like the pattern of prepending `Z` to shared components. Can we work around it? If not, I think `ZulipButton` is clearer